test/cosmoz-bottom-bar-view.html

```diff
@@ -98,10 +98,65 @@
 					assert.isFalse(bottomBar.visible);
 					done();
 				}, 50);
+			});
+
+			test('bottomBar is visible when scrolling up', function (done) {
+				bottomBarView.active = false;
+				bottomBarView.$.scroller.scrollTop = 10;
```

Comment:
I think the test should scroll down, wait, scroll up and then assert so the tests doesn't have any relationship to each other.

Reply:
Added nested async calls. I can make them unnested if I raise the timeouts, but that will run the timeouts simultaneously, not sure if that is better.